erpnext/public/js/controllers/accounts.js

```diff
@@ -246,6 +248,24 @@ if(!erpnext.taxes.flags[cur_frm.cscript.tax_table]) {
 		cur_frm.cscript.validate_taxes_and_charges(cdt, cdn);
 	});
 
+	frappe.ui.form.on(cur_frm.cscript.tax_table, "category", function(frm, cdt, cdn) {
+		if(open_form) {
```

Comment:
Running the trigger causes the following issue: `ReferenceError: open_form is not defined`